.cirrus.yml

```diff
@@ -99,7 +103,9 @@ osx_task:
 
 cross_task:
     alias: cross
-    only_if: *not_docs_multiarch
+    only_if: |
```

Comment:
(I have no idea if the difference between `|` and `>-` matters here, just noting the change.)

Reply:
Doesn't matter much in this context.  Both are valid but I'll put back the old one since it's slightly preferred.

FWIW: `>-` means "fold" the next indented block into a single-line string, replacing newlines with ' '.  Whereas `|` does the same thing, but keeps the newlines.